frontend/packages/core/src/Network/index.ts

```diff
@@ -25,6 +25,23 @@ const successInterceptor = (response: AxiosResponse) => {
   return response;
 };
 
+const successProxyInterceptor = (response: AxiosResponse) => {
+  // Handle proxy errors
+  if (response.data.httpStatus >= 400) {
+    const error = {
+      status: {
+        code: response.data.httpStatus,
+        text: HTTP_CODE_MAPPING[response.data.httpStatus] || "Unknown",
```

Comment:
Maybe we could create a function that return the right HTTP code status or "Unknown", just to avoid use `"|| Unknown"` everytime.